### PR TITLE
feat: add possibility to ignore username errors on first login screen

### LINKED ISSUE #3383: Return Error after Password Check
The organisations should be able to set an attribute to not return an error message if the user is not found

**Acceptance criteria**
- [x] Extend LoginPolicy with CheckUserAfterPassword
- [x] Extend LoginPolicy in console
- [x] Change Login Flow
- [x] If no user found don't show error message, go to password screen
- [x] Generic Error message (Username or password is wrong)
- [x] If This error message is shown, the username screen will be shown again (not the passwordscreen)

### PR DESCRIPTION
**Pull request by livio-a**
closes #3383

### CODE REVIEW

**Review by fgerschwiler — COMMENTED** [review]
custom login texts missing

**Review by fgerschwiler — APPROVED** [review]

File: console/src/assets/i18n/it.json [hunk 1]
```diff
@@ -991,6 +991,8 @@
       "FORCEMFA_DESC": "Se l'opzione \u00e8 selezionata, gli utenti devono configurare un secondo fattore per il login.",
       "HIDEPASSWORDRESET": "Nascondi ripristino della password",
       "HIDEPASSWORDRESET_DESC": "Se l'opzione \u00e8 selezionata, l'utente non pu\u00f2 resettare la sua password nel interfaccia login.",
+      "IGNOREUNKNOWNUSERNAMES": "Ignora i nomi utente sconosciuti",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:994]:
```suggestion
      "IGNOREUNKNOWNUSERNAMES": "Ignora un nome utente sconosciuto",
```

File: console/src/assets/i18n/it.json [hunk 2]
```diff
@@ -991,6 +991,8 @@
...
+      "IGNOREUNKNOWNUSERNAMES_DESC": "Se l'opzione \u00e8 selezionata, il passo della password viene visualizzato nel login anche se l'utente non \u00e8 stato trovato. All'utente non viene mostrato se il nome utente o la password erano errati durante il controllo della password",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:995]:
```suggestion
      "IGNOREUNKNOWNUSERNAMES_DESC": "Se l'opzione \u00e8 selezionata, l'inserimento della password viene mostrato anche se nessun utente è stato trovato. Nota che dopo il controllo della password, non viene mostrato se il nome utente o la password erano errati.",
```

File: console/src/assets/i18n/en.json [hunk 3]
```diff
@@ -991,6 +991,8 @@
       "FORCEMFA_DESC": "If the option is selected, users have to configure a second factor for login.",
       "HIDEPASSWORDRESET": "Hide Password reset",
       "HIDEPASSWORDRESET_DESC": "If the option is selected, the user can't reset his password in the login process.",
+      "IGNOREUNKNOWNUSERNAMES": "Ignore unknown usernames",
+      "IGNOREUNKNOWNUSERNAMES_DESC": "If the option is selected, the password screen will be display in the login process even if the user was not found. The error on the password check will not disclose if the username or password was wrong.",
```
Comment (peintnermax) [inline console/src/assets/i18n/en.json:995]:
```suggestion
      "IGNOREUNKNOWNUSERNAMES_DESC": "If the option is selected, the password screen will be displayed in the login process even if the user was not found. The error on the password check will not disclose if the username or password was wrong.",
```

**Review by peintnermax — APPROVED** [review]

**Pull request merged**